test/helpers/cilium.go

```diff
@@ -118,7 +120,8 @@ func (c *Cilium) EndpointSetConfig(id, option, value string) bool {
 	return true
 }
 
-var EndpointWaitUntilReadyRetry int = 0 //List how many retries EndpointWaitUntilReady should have
+// EndpointWaitUntilReadyRetry List how many retries EndpointWaitUntilReady should have
```

Comment:
Same as above; please use complete sentences with comments that document publicly-facing constructs.